src/Popover/Popover.js

```diff
@@ -139,12 +151,17 @@ export default class Popover extends Component<Props, State> {
     };
     const contentId = `${this.id}-popoverContent`;
 
+    const child = Children.only(children);
+
     let popoverTriggerProps = {
       contentId,
-      children,
+      children: child,
       disabled,
-      isOpen,
-      onClick: !disabled ? this.toggleOpenState : undefined,
+      isOpen: Boolean(isOpen),
+      onClick: !disabled
+        ? // TODO: is this ok to assume/force one child?
+          composeEventHandlers(this.toggleOpenState, child.props.onClick)
```

Comment:
The question is how do we give the user a way to prevent `toggleOpenState` from being called on button click.  Is it important?

`composeEventHandlers` somewhat allows for this with `defaultPrevented`

You could try reordering these functions, then try with a child whose `onClick` uses `event.preventDefault()`

Reply:
Another simpler option might be to only `toggleOpenState` when there is no `onClick`.  e.g.

```js
onClick: child.props.onClick || this.toggleOpenState
```

I wold do some experimentation here and see what feels right.

Reply:
Perhaps I am overthinking this one.  Let's leave it as is for now.